### PR TITLE
Fix examples in subresource-loading

### PR DESCRIPTION
**Pull request by hayatoito**
The keys in Json should be quoted.

### CODE REVIEW

File: explainers/subresource-loading.md [hunk 1]
```diff
@@ -249,8 +249,8 @@ In the following, `a.js` will be loaded, but `b.js` will be blocked:
 ```
 <script type="webbundle">
 {
-  source: "https://example.com/subresources.wbn",
-  resources: ["https://example.com/script/a.js",
+  "source": "https://example.com/subresources.wbn",
+  "resources": ["https://example.com/script/a.js",
               "https://example.com/b.js"]
```
Comment (irori) [inline explainers/subresource-loading.md:254]: nit: Add 2 more spaces (to maintain indent)

File: explainers/subresource-loading.md [hunk 2]
```diff
@@ -266,8 +266,8 @@ Instead of including a list of resources, the `<script>` defines a `scopes`.
 ```html
 <script type="webbundle">
 {
-  source: "https://example.com/dir/subresources.wbn",
-  scopes: ["https://example.com/dir/js/",
+  "source": "https://example.com/dir/subresources.wbn",
+  "scopes": ["https://example.com/dir/js/",
            "https://example.com/dir/img/",
            "https://example.com/dir/css/"]
```
Comment (irori) [inline explainers/subresource-loading.md:272]: ditto.

**Review by irori — APPROVED** [review]
LGTM with nits

File: explainers/subresource-loading.md (hunk 1 above)
Comment (hayatoito) [reply explainers/subresource-loading.md]: Done.

File: explainers/subresource-loading.md (hunk 2 above)
Comment (hayatoito) [reply explainers/subresource-loading.md]: Done.

**Pull request merged**